rust/relay/src/main.rs

```diff
@@ -160,6 +160,8 @@ async fn main() -> Result<()> {
             }
         }
     } else {
+        tracing::info!("No portal token supplied, starting standalone mode");
```

Comment:
Let's make this a warn

Reply:
Not sure I agree. There is nothing fundamentally wrong with starting without a token and I'd expect that logging on info is the default so one would definitely see this message.

Reply:
Merged it because I think people that are deploying the relay don't have enough context to easily notice this and it can be a head-scratching problem.